proposals/interledger/index.html

```diff
@@ -30,37 +30,31 @@
                   title:    "The Interledger Protocol"
                   ,   href:     "https://interledger.org/rfcs/0003-interledger-protocol/draft-9.html"
                   ,   authors:  [
-                      "Evan Schwartz",
-                      "Stefan Thomas",
-                      "Ben Sharafian"
+                      "Interledger Community"
```

Comment:
would this not be the publisher instead of the author?

Reply:
@michielbdejong since we have not tracked the editors on these docs directly I have used the committers as the authors list

Reply:
If you have to name individual community members, then looking at https://github.com/interledger/rfcs/commits/master/0003-interledger-protocol/0003-interledger-protocol.md, for e.g. IL-RFC-3 they would be, in reverse chronological order, Stefan, me, Jason, Andrew, you, Sylvain, and Evan (not Ben).
For IL-RFC-15, looking at https://github.com/interledger/rfcs/commits/master/0015-ilp-addresses/0015-ilp-addresses.md, it would be (again in reverse chronological order): me, you, and Rome (not Stefan, not Evan).
For IL-RFC-26 you would need to look at https://github.com/interledger/rfcs/commits/master/0026-payment-pointers/0026-payment-pointers.md - it would be just you and not Ben.

So which committers did you look at, @adrianhopebailie? I get at least different results than you did. And even then, I think we should not pretend that Interledger was "authored" by whoever updates the RFCs, it sidelines a lot of other people, and doesn't do right to our community spirit?

@marcoscaceres Is it a hard requirement of w3c documents that references name a list of specific humans as authors? Can this document not reference any community-authored documents, in the way I proposed?

Reply:
@michielbdejong as I said, this may be wrong as I haven't updated those for some time.

> And even then, I think we should not pretend that Interledger was "authored" by whoever updates the RFCs, it sidelines a lot of other people, and doesn't do right to our community spirit?

I agree with this sentiment, I hadn't really thought about it that way when the committers list was still small.

Let me investigate some options